stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/MessageText.kt

```diff
@@ -28,11 +35,13 @@ import io.getstream.chat.android.compose.ui.theme.ChatTheme
  *
  * @param message Message to show.
  * @param modifier Modifier for styling.
+ * @param onLongItemClick Handler used for long pressing on the message text.
  */
 @Composable
 public fun MessageText(
     message: Message,
     modifier: Modifier = Modifier,
+    onLongItemClick: (Message) -> Unit,
```

Comment:
This change is currently breaking in case anyone is using `MessageText` directly.
However I made this purposefully taking into consideration 2 thing.

1. We have a chance to do this before stable
2. It signals the issue to the customers and hopefully triggers them to read the release notes

It could easily be made non-breaking by introducing a default argument but I was worried that this would only hide the issue for our customers and not make them address it if they have it.

Also this isn't my favorite solution to the issue. I wanted to backpropagate events to the parent composable but doing that would require too much low level work as unfortunately `Modifier.pointerInput()` does not allow signaling that pointer events were not consumed.

Reply:
Sounds good - make sure to add it to the changelog as a "change"